src/components/Send.jsx

```diff
@@ -502,7 +479,7 @@ export default function Send() {
                     <td>{t('send.sweep_amount_breakdown_total_balance')}</td>
                     <td className={styles['balance-col']}>
                       <Balance
-                        valueString={breakdown.totalBalance.toString()}
+                        valueString={accountBalanceOrNull.totalBalance}
```

Comment:
It's a bit confusing maybe that `totalBalance` is a string and `calculatedFrozenOrLockedBalanceInSats` below is a number. Would it be easy to make all these balances the same type? If not then it's probably not a big deal if we just keep it as is.

Reply:
Yes, it is a bit confusing, I am sorry.
No, unfortunately this is not that easy or better put: It would need much additional work and _testing_. I refrained from doing so as this should just be fixing the format change issue without touching too much other code.

Is it okay to handle the string/number and representation issue in a tailored issue?
It needs improvements in general, e.g. the methods `btcToSats` and `satsToBtc` have floating point issues.

e.g.
```ts
const satsToBtc = (value: string) => parseInt(value, 10) / 100000000

> satsToBtc("1")
> // outputs: 1e-8
```

Reply:
👍 Yeah let's do it like this then!